converters/fromZigbee.js

```diff
@@ -3658,6 +3658,21 @@ const converters = {
             }
         },
     },
+    tuya_dimmer: {
+        cluster: 'manuSpecificTuyaDimmer',
+        type: 'commandGetData',
```

Comment:
I think this command also has to be added to https://github.com/Koenkk/zigbee-herdsman/blob/master/src/controller/events.ts#L30 (https://github.com/Koenkk/zigbee-herdsman/pull/125)

Reply:
Yes, I forgot this block. Added